__tests__/middleware-test.js

```diff
@@ -0,0 +1,158 @@
+jest.mock('@agoric/casting', () => {
+    return {
+        iterateReverse: jest.fn(),
+        makeLeader: jest.fn(),
+        makeFollower: jest.fn(),
+    };
+});
+
+jest.mock('../src/lib/rpc.js', () => {
+    return {
+        boardSlottingMarshaller: jest.fn(),
+        networkConfig: { rpcAddrs: ["http://127.0.0.1:26657"], chainName: "agoriclocal" },
+        makeAgoricNames: jest.fn()
+    };
+});
+
+jest.mock('../src/lib/wallet.js', () => {
+    return {
+        getCurrent: jest.fn(),
+    };
+});
+
+jest.mock('../src/oracle/helper.js', () => {
+    const originalModule = jest.requireActual("../src/oracle/helper")
+    return {
+        ...originalModule,
+        readVStorage: jest.fn()
+    };
+});
+
+import { readJSONFile, readVStorage } from '../src/oracle/helper.js';
+import { getOffers, checkSubmissionForRound, queryPrice, queryRound, getOraclesInvitations } from '../src/oracle/middleware';
```

Comment:
```suggestion
import {
    getOffers,
    checkSubmissionForRound,
    queryPrice,
    queryRound,
    getOraclesInvitations
} from '../src/oracle/middleware';
```

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/2a7bc45b4dfcdcca162fb6a799c0d6ccffe75de5